reana-ui/src/actions.js

```diff
@@ -78,7 +78,9 @@ const USER_SIGNOUT_URL = `${api}/api/logout`;
 const USER_REQUEST_TOKEN_URL = `${api}/api/token`;
 const USER_CONFIRM_EMAIL_URL = `${api}/api/confirm-email`;
 const WORKFLOWS_URL = (params) =>
-  `${api}/api/workflows?verbose=true&${stringifyQueryParams(params)}`;
+  `${api}/api/workflows?verbose=true&include_progress=true&include_workspace_size=true&${stringifyQueryParams(
```

Comment:
Shouldn't `verbose` be backward-compatible? Ideally this PR shouldn't be needed at all.